jarvis.py

```diff
@@ -2438,8 +2438,7 @@ def volume_controller(level):
 
 def face_recognition_detection():
     """Initiates face recognition script and looks for images stored in named directories within 'train' directory."""
-    if operating_system == 'Darwin':
-        sys.stdout.write("\r")
+    sys.stdout.write("\r")
```

Comment:
Hey bud, your indentation is off for lines starting 2442 to 2490
Can you fix that and pull the latest branch from master? Guess my commits later on are causing conflicts in files.